cpp/oneapi/dal/algo/kmeans/backend/cpu/train_kernel_lloyd_dense.cpp

```diff
@@ -83,63 +83,106 @@ static daal::data_management::NumericTablePtr get_initial_centroids(
     return daal_initial_centroids;
 }
 
+inline auto get_daal_parameter_to_train(const descriptor_t& desc) {
+    const std::int64_t cluster_count = desc.get_cluster_count();
+    const std::int64_t max_iteration_count = desc.get_max_iteration_count();
+    const double accuracy_threshold = desc.get_accuracy_threshold();
+
+    daal_kmeans::Parameter par(dal::detail::integral_cast<std::size_t>(cluster_count),
+                               dal::detail::integral_cast<std::size_t>(max_iteration_count));
+
+    par.accuracyThreshold = accuracy_threshold;
```

Comment:
I guess this can't be set during parameter construction?

Reply:
yes, there is no set function for accuracyThreshold